src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
```

Comment:
Ensure that the element with the class 'spider' exists in the DOM before setting its style properties. If it doesn't exist, `spider` will be `null`, and attempting to set `style.top` or `style.left` will result in an error.